sdk/anomalydetector/Azure.AI.AnomalyDetector/samples/Sample4_MultivariateDetect.md

```diff
@@ -0,0 +1,205 @@
+# Detect Multivariate anomaly
+This sample shows how to detect the anomalies in multivariate time series.
+
+To get started, make sure you have satisfied all the prerequisites and got all the resources required by [README][README].
+
+## Create an AnomalyDetectorClient
+
+To create a new `AnomalyDetectorClient` you need the endpoint and credentials from your resource. In the sample below you'll use an Anomaly Detector API key credential by creating an `AzureKeyCredential` object.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateAnomalyDetectorClient
+//read endpoint and apiKey
+string endpoint = TestEnvironment.Endpoint;
+string apiKey = TestEnvironment.ApiKey;
+
+var endpointUri = new Uri(endpoint);
+var credential = new AzureKeyCredential(apiKey);
+
+//create client
+AnomalyDetectorClient client = new AnomalyDetectorClient(endpointUri, credential);
+```
+
+## Train the model
+
+Create a new private async task as below to handle training your model. You will use `TrainMultivariateModel` to train the model and `GetMultivariateModelAysnc` to check when training is complete.
+
+You could add the data source, along with start time and end time to a `ModelInfo` object to create a data feed. The data source is a shared access signature(SAS) link in the format https://\[placeholder\]/. To generate the datasource link, you could first download our [sample data][datasource], then upload it to a azure container according to the [Upload a block blob][upload_blob] documentation and get the SAS link of the data according to the [Create SAS tokens for blobs in the Azure portal][generate_sas] documentation.
+
+Call `TrainMultivariateModel` with the created data feed and extract the model ID from the response header. Afterwards, you can get the model info, including the model status, by calling `GetMultivariateModelAsync` with the model ID . Wait until the model status is ready. 
+
+```C# Snippet:TrainModel
```

Comment:
Snippet region name is different. It should be the same as `TrainMultivariateModel`

Reply:
Please fix this for other snippets as well.